Regression/prepare_file_travis.py

```diff
@@ -67,9 +67,10 @@
     text = re.sub('addToCompileString =',
                   'addToCompileString = USE_CCACHE=TRUE ', text)
 
-# Add runtime option: crash for unused variables
+# Add runtime options: crash for unused variables; trap NaNs, divisions by zero, and overflows
 text = re.sub('runtime_params =',
-              'runtime_params = amrex.abort_on_unused_inputs=1 ',
+              'runtime_params = amrex.abort_on_unused_inputs=1 ' + 'amrex.fpe_trap_invalid=1 ' +
+              'amrex.fpe_trap_zero=1 ' + 'amrex.fpe_trap_overflow=1 ',
```

Comment:
Maybe with less `+`s like this?
```suggestion
              'runtime_params = amrex.abort_on_unused_inputs=1 amrex.fpe_trap_invalid=1 ' +
              'amrex.fpe_trap_zero=1 amrex.fpe_trap_overflow=1 ',
```

Reply:
Thanks, done. I also removed the last trailing white space, which was not necessary.